README.md

```diff
@@ -2,6 +2,28 @@
 
 *Say the magic word.*
 
+`plz` is a job runner targetted at training machine learning models as cheaply as possible.
+It is, however, fairly general, so can be used for practically anything that requires running a job.
```

Comment:
'a job in the cloud'? 'a job in a repeatable fashion'? The description should give you an idea of what it does for you (beyond 'running a job', which is not very informative)